x/cosmwasm/README.md

```diff
@@ -0,0 +1,34 @@
+# CosmWasm support
```

Comment:
I had some discussions about naming (not sure who) and that this should not be a module (under x) as we don't fulfil the module interface.

There is currently [app/wasmtest](https://github.com/confio/osmosis/tree/main/app/wasmtest) with some test code, and I would propose renaming that to `app/wasm` or `app/cosmwasm` and adding your work to that package. (Those test cases are also great starting points to "cut and paste" and make one for a custom contract)

Reply:
We discussed this with Alex, but wasn't sure where to put this anyway. Thanks.